XPlaneReflection/XPlaneReflectionTests.c

```diff
@@ -0,0 +1,99 @@
+#include "XPlaneReflectionTests.h"
+#include "XPlaneReflection.h"
+
+/* 
+ Tests if the x value was correctly flipped
+ Returns 1 if successful, 0 otherwise
+*/
+int didXReflect( struct point * oldP, struct point * newP ) {
+	if( oldP->element[0] == -newP->element[0]) {
+		return 1;
+	}
+	else {
+		fprintf( stderr, "old point x-value: %f, new point x-value: %f\n", oldP->element[0], newP->element[0] );
+		fprintf( stderr, "Error: X value did not reflect correctly. Exiting the program.\n" );
+		exit(1);
+	}
+}
+
+/*
+ Tests if the other values in the point are unchanged
+ Returns 1 if successful, 0 otherwise
+*/
+int testOtherValuesUnchanged( struct point * oldP, struct point * newP ) {
+	for( int i = 1; i < 4; i++ ) {
+		if( oldP->element[i] != newP->element[i]) {
+		    fprintf( stderr, "old point x-value: %f, new point x-value: %f\n", oldP->element[i], newP->element[i] );
+		    fprintf( stderr, "Error: the values are changed. Exiting the program.\n" );
+		    exit(1);
```

Comment:
although tests pass just wondering if having the tests exit is a good idea as it may limit which tests you know specifically failed. this would also prevent all other tests from running.

Reply:
Yea you're right, definitely better to return 0 than to exit. Ill make that change rn